doc/Command-Reference.md

```diff
@@ -5121,6 +5121,50 @@ This command is to display the link-training status of the selected interfaces.
     Ethernet8      trained          on      up       up
   ```
 
+**show interfaces mac-statistics**
+
+The show interface mac-stats command provides detailed statistics and error counters for MAC-level operations on an interface. It displays the status of various operational parameters, error counts, and timestamps for when these errors occurred.
+
+- Usage:
+  ```
+  show interfaces mac_statistics [<interface_name>]
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ show interfaces mac_statistics Ethernet4
```

Comment:
Suggest to have the header as follows:
`Port Errors         Count           Last timestamp(UTC)`

Reply:
If the error count is 0, the display timestamp as `Never`